Tests-Legacy/L0/XamariniOS/_suite.ts

```diff
@@ -212,24 +181,17 @@ describe('XamariniOS Suite', function() {
         tr.setInput('args', '');
         tr.setInput('packageApp', ''); //boolean
         tr.setInput('forSimulator', ''); //boolean
-        tr.setInput('buildTool', 'xbuild');
-        tr.setInput('mdtoolLocation', '/user/bin/');
-        tr.setInput('unlockDefaultKeychain', ''); //boolean
-        tr.setInput('defaultKeychainPassword', '');
-        tr.setInput('p12', ''); //path
-        tr.setInput('p12pwd', '');
+        tr.setInput('buildToolLocation', '/user/bin/');
         tr.setInput('iosSigningIdentity', '');
         tr.setInput('provProfileUuid', '');
-        tr.setInput('provProfile', ''); //path
-        tr.setInput('removeProfile', ''); //boolean
         
         tr.run()
         .then(() => {
             assert(tr.invokedToolCount == 0, 'should not have run XamariniOS');
             assert(tr.resultWasSet, 'task should have set a result');
             assert(tr.stderr.length > 0, 'should have written to stderr');
             assert(tr.failed, 'task should have failed');
-            assert(tr.stderr.indexOf('not found build tool: /user/bin/xbuild') >= 0, 'wrong error message');            
+            assert(tr.stderr.indexOf('not found build tool: /user/bin/msbuild') >= 0, 'wrong error message');            
```

Comment:
Can use `contains`.  And since `tr.stderr` is a string, we might as well print out what the actual `stderr` was in the diagnostic message.